packages/playwright-ct-angular/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@playwright/experimental-ct-angular",
+  "version": "1.40.0-next",
+  "description": "Playwright Component Testing for Angular",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/sand4rt/playwright-ct-angular.git"
+  },
+  "homepage": "https://playwright.dev",
+  "engines": {
+    "node": ">=16"
+  },
+  "author": {
+    "name": "Microsoft Corporation"
+  },
+  "license": "Apache-2.0",
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "default": "./index.js"
+    },
+    "./register": {
+      "types": "./register.d.ts",
+      "default": "./register.mjs"
+    },
+    "./hooks": {
+      "types": "./hooks.d.ts",
+      "default": "./hooks.mjs"
+    }
+  },
+  "dependencies": {
+    "@analogjs/vite-plugin-angular": "0.2.10",
+    "@angular-devkit/build-angular": "^16.1.0",
+    "@playwright/experimental-ct-core": "1.40.0-next",
+    "vite": "^4.4.9"
+  },
+  "devDependencies": {
+    "@angular/animations": "^16.1.7",
```

Comment:
shouldn't all `@angular/*` be peerDeps?

Reply:
and we can remove:
`@angular/animations`
`@angular/common`
`@angular/compiler`
`@angular/compiler-cli`
`@angular/platform-browser`
as they are not used

Reply:
What about making peer dependencies range version compatibility more strict by using something like `^15.0.0 || ^16.0.0` instead of `>=15.1.0 || >=16.0.0`? The problem with `>=` is that we allow any greater majors, but in reality, there are breaking changes so we cannot guarantee compatibility.

Reply:
Good one @edbzn!